content-repo/extra-docs/articles/integrations-and-incident-health-check.md

```diff
@@ -0,0 +1,69 @@
+| Integrations and Playbooks Health Check - Running Scripts | This playbook is triggered by a 'JOB - Integrations and Playbooks Health' playbook responsible for running failed integrations and failed incidents scripts. The playbook may run separately from the main playbook to run health tests on enabled integrations and open incidents | This sub-playbook is **responsible for running the health check scripts** |
+| JOB - Integrations and Playbooks Health Check - Lists handling | This playbook is triggered by a 'JOB - Integrations and Playbooks Health' playbook and is responsible for creating or updating related XSOAR lists | This sub-playbook is **responsible for save the scripts data to XSOAR list** |
+ ---
+ 
+Besides running the sub-playbooks, The master playbook "JOB - Integrations and Playbooks Health Check" will also:
+Send the health check report to the recipients from the playbook inputs.
+If another health check investigation is open (incident nor a job):
+The playbook will **check if analyst notes were added** to the failed integrations and failed incidents grids and **copy** them to the new investigation.
+The playbook will link the old investigation to the new one and close the old investigation due to irrelevance.
+Re-run the health check tests. It is a manual action because you will want to re-run them only after fixing the issues and not before.
+ 
+### Incident Layouts
+The incident type contains two layouts: one for failed integrations and the other for failed incidents.
+In the failed integrations layout, we see the number of failed instances, their category, the number of totals checked (AKA enabled) instances, and the grid that contains details with why they failed. 
+The column 'Analyst Notes' in the grid is for **manual** use for tracking and adds comments about the integration status.
+The same for failed incidents layout, we have the number of failed incidents, the total number of errors, the number of unassigned incidents (AKA incidents that failed, and no one knows because no one assigned for those incidents). We also have here the top failed commands and playbook names that failed. And the grid as you saw in the integrations layout with the analyst notes.
+ 
+Note: When the master playbook copies the analyst note from the integration \ incident to the new investigation, the old investigation's date is added.
+ 
+### Dashboards and Lists
+The dashboard widgets use the data from the lists created or updated by the playbook.
+The dashboard **isn't** a real-time view **The dashboard displays are the data for the time the playbook was running.**
+For example, if the job/playbook runs once a day, the dashboard data will be updated once a day (at the same time the playbook is running).
+ 
+## Before You Start
+ 
+The playbooks in the pack use two scripts to check the integrations and incidents. 
+"Demisto REST API" integration is mandatory for one of the scripts to run.
```

Comment:
The Demisto REST API pack will be automatically installed with this pack if not already installed.